tests/phpunit/tests/image/functions.php

```diff
@@ -1152,4 +1152,32 @@ public function data_wp_exif_frac2dec() {
 			),
 		);
 	}
+
+	/**
+	 * Test that PHP and underlying external GD library actually supports AVIF.
+	 *
+	 * @link https://github.com/php/php-src/issues/12019
+	 *
+	 * @ticket 60628
+	 */
+	public function test_php_avif_support() {
+		if ( imagetypes() & IMG_AVIF ) {
+			$test_file = get_temp_dir() . 'php-gd.avif';
+
+			$result = @imageavif( imagecreatetruecolor( 16, 16 ), $test_file );
```

Comment:
`Call to undefined function imageavif()` the current setup didn;t support avif and in most avif test we skipped so there may be issue with the Lib. where the support is not working.